stests/core/domain/enums.py

```diff
@@ -32,6 +32,16 @@ class BlockStatus(enum.Flag):
     REJECTED = enum.auto()
 
 
+class ClientContractType(enum.Enum):
+    """Enumeration over set of bundled client contract types.
+    
+    """
+    STD_PAYMENT =  "standard_payment.wasm"
+    TRANSFER = "transfer_to_account.wasm"
```

Comment:
I think `transfer_to_account.wasm` is not used anymore, it has been replaced with `transfer_to_account_u512.wasm`.

Reply:
OK thx - removed.